File: string-paradigm-expression-no-wrapper/src/main/java/com/jesperancinha/string/paradigms/nowrapper/expression/SpellingCheck.java
package com.jesperancinha.string.paradigms.nowrapper.expression;

import java.util.List;

public class SpellingCheck extends SpellingCheckBase {


	@SuppressWarnings("unchecked")
	public List<String> getSpellingDoubleCommaRegions() {
		return (List<String>)(List<?>)spellingDoubleCommaRegions;
	}

	@SuppressWarnings("unchecked")
	public List<String> getSpellingWhiteSpacesRegions() {
		return (List<String>)(List<?>)spellingWhiteSpacesRegions;
	}

	@SuppressWarnings("unchecked")
	public List<String> getSpellingBeginCapitalRegions() {
		return (List<String>)(List<?>)spellingBeginCapitalRegions;
	}
}

File: string-paradigm-expression-no-wrapper/src/main/kotlin/org/jesperancinha/string/paradigms/nowrapper/expression/SpellingCheck.kt
package org.jesperancinha.string.paradigms.nowrapper.expression

class SpellingCheck : SpellingCheckBase() {
    override val spellingDoubleCommaRegions: List<String>?
        get() = spellingDoubleCommaRegions as List<*> as List<String>
    override val spellingWhiteSpacesRegions: List<String>?
        get() = spellingWhiteSpacesRegions as List<*> as List<String>
    override val spellingBeginCapitalRegions: List<String>?
        get() = spellingBeginCapitalRegions as List<*> as List<String>
}